src/app/components/ManualEntry.svelte

```diff
@@ -42,39 +83,46 @@
     box-shadow: 0 25px 40px 0 rgba(0, 0, 0, 0.1);
   }
 
-  tr {
+  table th {
+    vertical-align: middle;
+    padding: 1.25rem 0 1.25rem 1.25rem;
+    background-color: var(--lite-white);
+    text-align: center;
+  }
+
+  table tr {
     border-bottom: 1pt solid var(--lite-white);
   }
 
-  td {
+  table td {
     vertical-align: middle;
     padding: 0.625rem 0 0.625rem 1.25rem;
     color: var(--gray-dark);
     text-align: center;
   }
 
-  th:first-of-type {
+  table th:first-of-type {
     border-top-left-radius: 10px;
   }
-  th:last-of-type {
+  table th:last-of-type {
     border-top-right-radius: 10px;
   }
-  tr:last-of-type td:first-of-type {
+  table tr:last-of-type td:first-of-type {
     border-bottom-left-radius: 10px;
   }
-  tr:last-of-type td:last-of-type {
+  table tr:last-of-type td:last-of-type {
     border-bottom-right-radius: 10px;
   }
 
-  .btn--inline {
+  table .btn--inline {
     display: inline;
     padding: 0;
     border: 0;
     background-color: transparent;
     font-size: 1em;
   }
 
-  td > input {
+  table td > input {
     height: 20px;
```

Comment:
Use `rem` units for this measure, so that it scales with the containing font-size